playbooks/cluster_setup_auditd.yaml

```diff
@@ -0,0 +1,17 @@
+# Copyright (C) 2023, RTE (http://www.rte-france.com)
```

Comment:
I think this would make more sense to be integrated in the debian hardened role.

Reply:
All tasks have been moved in debian hardening role.